src/main/java/org/dcsa/ebl/Application.java

```diff
@@ -1,26 +1,29 @@
 package org.dcsa.ebl;
 
+import com.fasterxml.jackson.annotation.JsonInclude;
 import com.fasterxml.jackson.databind.DeserializationFeature;
 import com.fasterxml.jackson.databind.ObjectMapper;
-import org.dcsa.core.repository.ExtendedRepositoryImpl;
 import org.springframework.boot.SpringApplication;
 import org.springframework.boot.autoconfigure.SpringBootApplication;
 import org.springframework.context.annotation.Bean;
-import org.springframework.context.annotation.ComponentScan;
-import org.springframework.data.r2dbc.repository.config.EnableR2dbcRepositories;
+
+import java.text.SimpleDateFormat;
 
 @SpringBootApplication
 public class Application {
 
-	@Bean
-	public ObjectMapper objectMapper() {
-		return new ObjectMapper()
-				.enable(DeserializationFeature.FAIL_ON_UNKNOWN_PROPERTIES)
-				.enable(DeserializationFeature.FAIL_ON_IGNORED_PROPERTIES)
-				.findAndRegisterModules();
-	}
+  @Bean
+  public ObjectMapper objectMapper() {
+    return new ObjectMapper()
+        .enable(DeserializationFeature.FAIL_ON_UNKNOWN_PROPERTIES)
+        .enable(DeserializationFeature.FAIL_ON_IGNORED_PROPERTIES)
+        .setSerializationInclusion(JsonInclude.Include.NON_NULL)
+        .setSerializationInclusion(JsonInclude.Include.NON_EMPTY)
+        .setDateFormat(new SimpleDateFormat("yyyy-MM-dd"))
+        .findAndRegisterModules();
+  }
```

Comment:
Should this be the default for all projects and if so, should we move it to DCSA-Event-Core or DCSA-Core instead in the long run?

Reply:
@nt-as  yes, I created a techdebt ticket for this: https://dcsa.atlassian.net/browse/DDT-909